### ISSUE TITLE
Explain different release build on README

### ISSUE THREAD

**Issue by mathieujobin** [phhusson/treble_experimentations#1449]
Hi, 

each releases comes with a lot of different download options.
it would be great for newbies like me to express what are the differences.
`gapps` is self-explanatory. `vanilla` is probably what is the most "bare bone"

but what is `go` and `floss` ? 

the other selections we need to make is between
`ab`, `aonly`
and again between `arm`, `arm32-binder64` and `arm64`
I am guess most new phones are `arm64`

thanks

**Comment by MXAN2CHA** [comment]
go maybe a lite version of gms and customized to run on low performance device,the full name may be "android go"
floss i remenber includes the replace of gms by some open sources apps
ab,and a  only depends on your device,you can check in a app "treble check",but remember it should always be ab you device is "system as root“，you can find out your arch in the app "treble check",too. 
check xda for more info,the repo is not a place to teach step by step

**Comment by MXAN2CHA** [comment]
i agree that we should explain the difference in"go""gapps""floss""vanilla"
although the author is mostly active in xda,i believe is better to be friendly to some user directly from github

**Comment by voltcraft1** [comment]
Ideally some one creates a PR so that https://github.com/penn5/TrebleCheck app suggests what ROMs are applicable for your device - in a nice table.

**Comment by bnee99** [comment]
> i agree that we should explain the difference in"go""gapps""floss""vanilla" although the author is mostly active in xda,i believe is better to be friendly to some user directly from github

thanks for explaining floss and go.
can you tell little about how when to use vndklite?
